tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_data_on_gcs_using_spark.py

```diff
@@ -0,0 +1,73 @@
+"""
+To run this code as a local test, use the following console command:
+```
+pytest -v --docs-tests -m integration -k "how_to_connect_to_data_on_gcs_using_spark" tests/integration/test_script_runner.py
+```
+"""
+
+# Python
+# <snippet name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_data_on_gcs_using_spark.py get_context">
```

Comment:
This does not need to be in a snippet as it's not used in the md.

Reply:
good catch @TrangPham, thank you. 

@alexsherstinsky would it be possible to remove the snippet tag before merging?

Reply:
@TrangPham Thank you -- I am fixing in the next change.  Good catch.